04_pig_latin/pig_latin.rb

```diff
@@ -0,0 +1,80 @@
+def translate(sentence)
+
+  sentence.split.map { |word|
+    cap = false
```

Comment:
Could you extract all these logic rules to a separate method and apply the method for every word?  e.g. `translate(word)` to return the formatted word